source/tutorials/pd-mice.rst

```diff
@@ -0,0 +1,749 @@
+
+.. download::
+   :url: https://data.qiime2.org/2019.4/common/gg-13-8-99-515-806-nb-classifier.qza
+   :saveas: gg-13-8-99-515-806-nb-classifier.qza
+
+It’s worth noting that Naive Bayes classifiers perform best when they’re trained for the specific hypervariable region amplified. You can train a classifier specific for your dataset based on the :doc:`training classifiers tutorial <feature-classifier>` or download classifiers for other datasets from the :doc:`QIIME 2 resource page <../data-resources>`. Classifiers can be re-used for consistent versions of the underlying packages, database and region of interest.
+
+.. command-block::
+
+   qiime feature-classifier classify-sklearn \
+     --i-reads ./deblur_rep_set.qza \
+     --i-classifier ./gg-13-8-99-515-806-nb-classifier.qza \
+     --o-classification ./taxonomy.qza
+
+.. do we want to throw clawback in here?
+
+Now, let’s review the taxonomy associated with the sequences using the ``qiime metadata tabulate`` function.
+
+.. command-block::
+
+   qiime metadata tabulate \
+     --m-input-file ./taxonomy.qza \
+     --o-visualization ./taxonomy.qzv
+
+Let’s also tabulate the representative sequences. Tabulating the representative sequences will allow us to see the sequence assigned to the identifier and interactively blast the sequence against the NCBI database.
+
+.. command-block::
+
+   qiime feature-table tabulate-seqs \
+     --i-data ./deblur_rep_set.qza \
+     --o-visualization ./deblur_rep_set.qzv
+
+.. question::
+
+   Find the feature, ``59196a586276f0be745d0e334fc071c6``. What is the taxonomic classification of this sequence? What’s the confidence for the assignment?
+
+   How many sequences are mapped to g__Akkermansia?
+
+   Use the tabulated representative sequences to look up these features. If you blast them against NCBI, do you get the same taxonomic identifier?
+
+
+Alpha Rarefaction and Selecting a Rarefaction Depth
+===================================================
+
+Although sequencing depth in a microbiome samples does not directly relate to the original biomass in a community, the relative sequencing depth has a large impact on observed communities (`Weiss et al, 2017`_). Therefore, for most diversity metrics, a normalization approach is needed.
+
+Current best practices suggest the use of rarefaction, a normalizational via sub sampling without replacement. Rarefaction occurs in two steps. First, samples which are below the rarefaction depth are filtered out of the feature table. Then, all remaining samples are subsampled without replacement to get to the sequencing depth. It’s both important and sometimes challenging to select a rarefaction depth for diversity analyses. Several strategies exist to figure out the right rarefaction depth, but alpha rarefaction is a data-driven way to approach the problem.
```

Comment:
```suggestion
Current best practices suggest the use of rarefaction, a normalizational via sub-sampling without replacement. Rarefaction occurs in two steps: first, samples which are below the rarefaction depth are filtered out of the feature table. Then, all remaining samples are subsampled without replacement to get to the sequencing depth. It’s both important and sometimes challenging to select a rarefaction depth for diversity analyses. Several strategies exist to figure out an appropriate rarefaction depth - we will primarily consider alpha rarefaction in this tutorial, because it is a data-driven way to approach the problem.
```